api/src/functions/graphql.js

```diff
@@ -24,4 +28,4 @@ export const handler = createGraphQLHandler({
     // Upgrade notes as of RedwoodJS 0.19.0, but this project does not use a db so we comment it out for now
     // db.$disconnect()
   }
-})
+});
```

Comment:
We expose a graphql wrapper function which first does session verification in it, and then passes the user to the actual `createGraphQLHandler` function. The implementation of this can be found: https://github.com/supertokens/supertokens-node/blob/master/lib/ts/redwood.ts#L97